_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -0,0 +1,161 @@
+---
+id: 24afc2
+name: Letter spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+	This rule checks that the `style` attribute is not used to prevent adjusting `letter- spacing` by using `!important`, except if it's at least `0.12` times the font size.
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][], for which the `style` attribute [declares][declared] the [letter-spacing][] CSS property.
+
+## Expectation
+
+For the test target, one of the following is true:
+
+- the [cascaded][] value of its [letter-spacing][] property, specified via the `style` attribute, does not have the [important flag][]; or
+- the [computed][] value of its [letter-spacing][] property is at least `0.12` times its [computed][] [font-size][].
+
+## Assumptions
+
+There is no mechanism available on the page to adjust letter spacing. If there is such a mechanism, failing this rule might not mean [success criterion 1.4.12 Text spacing](https://www.w3.org/TR/WCAG21/#text-spacing) is not satisfied.
+
+## Accessibility Support
+
+While some assistive technologies are able to set [user origin][] styles, others, such as browser extensions, are only able to set style with the [author origin][]. Such assistive technologies can not create styles with a high enough [priority][] to override a `style` attribute with the [important flag][]. If accessibility support does not include assistive technologies that override [letter-spacing][] through [author origin][], this rule should not be used.
+
+## Background
+
+- [Understanding Success Criterion 1.4.12: Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)
+- [CSS Text Module Level 3 - Spacing](https://www.w3.org/TR/css-text-3/#spacing)
+- [CSS Visual formatting model details](https://drafts.csswg.org/css2/visudet.html)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `div` element has a `letter-spacing` of `0.12em` which is equal to the recommended minimum given the specified font size is `1em`.
+
+```html
+<html>
+	<style>
+		body {
+			font-size: 1em;
+		}
+	</style>
+
+	<body>
+		<div style="letter-spacing: 0.12em !important">
+			The toy brought back fond memories of being lost in the rain forest.
+		</div>
+	</body>
+</html>
+```
+
+#### Passed Example 2
+
+This `p` element has a `letter-spacing` of `24px` specified via the style attribute, which is above to the recommended minimum given the specified font size is 16 pixels.
```

Comment:
Was it the goal to have a 24px letter spacing with a 16px font? The spacing is larger than the font :-) I'm not really sure that this is accessible to some people.
2px for letter spacing would be enough for passing the rule